vnext/package.json

```diff
@@ -69,5 +69,56 @@
       "minor",
       "patch"
     ]
-  }
+  },
+  "files": [
+    "/codegen",
+    "/Chakra",
+    "/Common",
+    "/DeforkingPatches",
+    "/etc",
+    "/flow",
+    "/Folly",
+    "/include",
+    "/jest",
+    "/JSI",
+    "/Libraries",
+    "!/Libraries/**/*.windesktop.*",
+    "/local-cli",
+    "/Microsoft.ReactNative",
+    "/Microsoft.ReactNative.Cxx",
+    "/Microsoft.ReactNative.Managed",
+    "/Microsoft.ReactNative.SharedManaged",
+    "/Mso",
+    "/PropertySheets",
+    "/ReactCommon",
+    "/ReactUWP",
+    "/ReactWindowsCore",
+    "/Scripts",
+    "/Shared",
+    "/stubs",
+    "/Test",
+    "/V8Inspector",
+
+    "/.clang-format",
+    "/.editorconfig",
+    "/.flowconfig",
+    "/Directory.Build.props",
+    "/Directory.Build.targets",
+    "/index.js",
+    "/index.windows.d.ts",
+    "/index.windows.d.ts.map",
+    "/index.windows.js",
+    "/index.windows.js.map",
+    "/interface.js",
+    "/just-task.js",
+    "/layoutFilesForNuget.bat",
```

Comment:
`layoutFilesForNuget.bat` can be removed

Reply:
Sounds good.

Reply:
Could not. The CI validation depends on it being in the package, so I'll leave it for now...
@acoates-ms: Should I file an issue for this?

Reply:
Do you know where? 

Our AZP publish pipeline calls https://github.com/microsoft/react-native-windows/blob/master/vnext/Scripts/Tfs/Layout-Headers.ps1 directly. The Github actions one (not currently working) uses the batch file, but should operate on source instead of a package from what I can tell?

Reply:
Yes file the issue.

Reply:
#4812